independent-publisher-connectors/Edatalia sign online/apiProperties.json

```diff
@@ -0,0 +1,43 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "endpoint":{
+        "type": "string",
+        "uiDefinition":{
+          "displayName": "Endpoint host",
+          "description": "Sandbox (restapi.firmar.info) Production (restapi.firmar.online)",
+          "tooltip": "Provide the end point host",
+          "constraints": {
+            "required": "true"
+          }
+        }
+      },
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#0058a0",
```

Comment:
@marpedroche all independent publishers must use "#da3b01" for their icon brand color. please see our templates [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json)